test/assets/messages/YouCannotAppealMessages.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package messages
+
+object YouCannotAppealMessages {
+  val title = "You cannot appeal yet - Appeal a VAT penalty - GOV.UK"
+
+  val headingPanelH1 = "You cannot appeal yet"
+
+  val agentLSPp1 = "You need to submit your client’s VAT Return before we can review this penalty."
+
+  val agentLSPp2 = "If you have already submitted the VAT Return, it can take up to 24 hours before it shows on your client’s VAT account."
+
+  val agentp3 = "If you need to cancel the VAT registration, you can do it through your client’s VAT details."
+
+  val agentLPPp1 = "Your client needs to pay their VAT bill before we can review this penalty."
+
+  val agentLPPp2 = "If they have already paid their VAT, it can take up to 5 days to clear to HMRC’s bank account and show on their payment history."
+
+  val returnToClientVATDetails = "Return to your client’s VAT details"
+
+  val returnToVATDetails = "Return to your VAT details"
+
+  val checkWhatYouOwe = "Check and pay what you owe"
+
+  val vatTraderLSPp1 = "You need to submit your VAT Return before we can review this penalty."
+
+  val vatTraderLSPp2 = "If you submitted the VAT Return very recently, it can take up to 24 hours before it shows on your VAT account."
+
+  val vatTraderLSPp3 = "If you need to cancel the VAT registration, you can do it through your VAT account."
```

Comment:
vatTraderLSPp3 and vatTraderLPPp3 are the same, maybe its worth just having one val for this?